genai_factory/src/genai_factory/workflows/workflow_server.py

```diff
@@ -11,14 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import logging
+from urllib.parse import urlparse
+
+import uvicorn
 
-from genai_factory import logger
 from genai_factory.config import WorkflowServerConfig
 from genai_factory.controller_client import ControllerClient
 from genai_factory.schemas import WorkflowType
 from genai_factory.sessions import SessionStore
 from genai_factory.workflows import Workflow
 
+# Initialize the MLRun-GenAI logger:
+logger = logging.getLogger("mlrun-genai-factory")
```

Comment:
```suggestion
logger = logging.getLogger("genai-factory")
```

also, move to the class init.